devtools/common/src/main/java/io/quarkus/cli/commands/file/GradleBuildFile.java

```diff
@@ -30,18 +30,6 @@
 
     public GradleBuildFile(ProjectWriter writer) throws IOException {
         super(writer, BuildTool.GRADLE);
-        if (writer.exists(SETTINGS_GRADLE_PATH)) {
```

Comment:
MavenBuildFile has similar forced initialization - should we remove that too in constructor ?

Reply:
That would probably make sense. Let me try it and see what happens

Reply:
Works!